modules/ztp-image-based-upgrade.adoc

```diff
@@ -3,10 +3,7 @@
 
 :_mod-docs-content-type: CONCEPT
 [id="ztp-image-based-upgrade-concept_{context}"]
-= Image-based upgrade for {sno} cluster with the {lcao}
-
-From {product-title} 4.14.7, the {lcao} 4.15 provides you with an alternative way to upgrade the platform version of a {sno} cluster.
-The image-based upgrade is faster than the standard upgrade method and allows you to directly upgrade from {product-title} <4.y> to <4.y+2>, and <4.y.z> to <4.y.z+n>.
+= Understanding the image-based upgrade for {sno} cluster with the {lcao}
 
 This upgrade method utilizes a generated OCI image from a dedicated seed cluster that is installed on the target {sno} cluster as a new `ostree` stateroot.
 A seed cluster is a {sno} cluster deployed with the target {product-title} version, Day 2 Operators, and configurations that is common to all target clusters.
```

Comment:
not touched by this PR, but just in case u'd like to consider it
```suggestion
A seed cluster is a {sno} cluster deployed with the target {product-title} version, Day 2 Operators, and configurations that are common to all target clusters.
```